### PR TITLE
docs(vital-layout): Add API docs and update docs for vital-layout

### PR DESCRIPTION
**Pull request by hvanyo**
## Changes

- Update API Docs
- Update Layout Docs
- Slight adjustments in vital-logo, copyright row and helmet tokens

## Test Instructions
* Smoke test on vital-logo (Default token was added)
* Smoke test on  copyrightrow (export was changed)
* Smoke test on vital-helmet (theme domain moved from base to default)
* Verify API doc : http://localhost:8000/___docs/#/Development/API/@bodiless/vital-layout/README
* Verify component doc : http://localhost:8000/___docs/#/VitalDesignSystem/Components/VitalLayout/

BREAKING CHANGE: 

* Rename of token  vitalHelmet.WithDesktopStatickBody -> vitalHelmet.WithDesktopStaticBody

### CODE REVIEW

File: packages/vital-layout/doc/README.md [hunk 1]
```diff
@@ -1,11 +1,22 @@
 # Vital Layout Components
 
-The Vital Layout (`@bodiless/vital-layout`) package is composed of the following components:
+The Vital Layout Package provides a set of components & tokens to compose the
+overall layout of a page. For more information on each componenent, visit the
```
Comment (dewen) [inline packages/vital-layout/doc/README.md:4]: componenent => component

File: packages/vital-layout/src/components/Footer/CopyrightRow/CopyrightRowClean.tsx [hunk 2]
```diff
@@ -39,11 +39,25 @@ const CopyrightRowCleanBase: FC<CopyrightRowProps> = ({ components: C, ...rest }
...
+ * with copyright & social links.
+ *
+ * @category Component
+ *
  */
-const CopyrightRowClean = designable(copyrightRowComponents, 'CopyrightRow')(CopyrightRowCleanBase);
+const CopyrightRowClean = withoutHydration()(CopyrightRowCleanDesignable);
 
+/**
+ * A token modifier that respects the CopyRightRow Compoments.
```
Comment (dewen) [inline packages/vital-layout/src/components/Footer/CopyrightRow/CopyrightRowClean.tsx:53]: Compoments => Components

File: packages/vital-layout/src/components/Footer/FooterClean.tsx [hunk 3]
```diff
@@ -57,10 +57,22 @@ const FooterCleanBase: FC<FooterProps> = ({ components: C, ...rest }) => (
...
- * A clean footer to be used in pages layouts.
+ * A clean footer to be used in pages layouts following vital design.
+ *
+ * @category Component
+ *
  */
 const FooterClean = designable(footerComponents, 'Footer')(FooterCleanBase);
 
+/**
+ * A token modifier that respects the Footer Compoments.
```
Comment (dewen) [inline packages/vital-layout/src/components/Footer/FooterClean.tsx:68]: Compoments => Components

File: packages/vital-layout/src/components/Footer/Rewards/RewardsClean.tsx [hunk 4]
```diff
@@ -49,10 +49,28 @@ const RewardsCleanBase: FC<RewardsProps> = ({ components: C, ...rest }) => (
...
+ * RECOMMEND TO NOT EXTEND/OVERRIDE and marked as deprecated.
+ *
+ * @category Component
+ * @deprecated
+ *
+ */
 const RewardsClean = designable(rewardsComponents, 'Rewards')(RewardsCleanBase);
 
+/**
+ * A token modifier that respects the Rewards Compoments.
```
Comment (dewen) [inline packages/vital-layout/src/components/Footer/Rewards/RewardsClean.tsx:64]: Compoments => Components

File: packages/vital-layout/src/components/Footer/CopyrightRow/CopyrightRowClean.tsx (hunk 2 above)
Comment (hvanyo) [inline packages/vital-layout/src/components/Footer/CopyrightRow/CopyrightRowClean.tsx:53]:
```suggestion
 * A token modifier that respects the CopyRightRow Components.
```

File: packages/vital-layout/src/components/Footer/FooterClean.tsx (hunk 3 above)
Comment (hvanyo) [inline packages/vital-layout/src/components/Footer/FooterClean.tsx:68]:
```suggestion
 * A token modifier that respects the Footer Components.
```

File: packages/vital-layout/src/components/Footer/Rewards/RewardsClean.tsx (hunk 4 above)
Comment (hvanyo) [inline packages/vital-layout/src/components/Footer/Rewards/RewardsClean.tsx:64]:
```suggestion
 * A token modifier that respects the Rewards Components.
```

File: packages/vital-layout/doc/README.md (hunk 1 above)
Comment (hvanyo) [inline packages/vital-layout/doc/README.md:4]:
```suggestion
overall layout of a page. For more information on each component, visit the
```

File: packages/vital-layout/src/components/Footer/SocialLinks/SocialLinksClean.tsx [hunk 5]
```diff
@@ -39,11 +39,27 @@ const SocialLinksCleanBase: FC<SocialLinksProps> = ({ components: C, ...rest })
...
+ * In future it will move to its own package and be a list of icons.
+ *
+ * @category Component
+ * @deprecated
+ *
+ */
 const SocialLinksClean = designable(socialLinksComponents, 'SocialLinks')(SocialLinksCleanBase);
 
+/**
+ * A token modifier that respects the Social Links Compoments.
```
Comment (hvanyo) [inline packages/vital-layout/src/components/Footer/SocialLinks/SocialLinksClean.tsx:55]:
```suggestion
 * A token modifier that respects the Social Links Components.
```

File: packages/vital-layout/src/components/Header/HeaderClean.tsx [hunk 6]
```diff
@@ -88,13 +88,32 @@ const HeaderCleanBase: FC<HeaderProps> = ({ components: C, ...rest }) => (
...
+/**
+ * Use this version of the header when all components are static.
+ *
+ * @category Component
+ *
+ */
 const HeaderStatic = withoutHydration()(HeaderClean);
 
+/**
+ * A token modifier that respects the Header Compoments.
```
Comment (hvanyo) [inline packages/vital-layout/src/components/Header/HeaderClean.tsx:107]:
```suggestion
 * A token modifier that respects the Header Components.
```

File: packages/vital-layout/src/components/Helmet/HelmetClean.tsx [hunk 7]
```diff
@@ -74,14 +74,21 @@ const helmetComponents: HelmetComponents = {
...
+ * @category Component
+ *
  */
 const HelmetClean = designable(helmetComponents, 'Helmet')(HelmetBase);
 
 export default HelmetClean;
 
 /**
- * Use this to define a token applicable to the VitalDS `HelmetClean` component.
+ * A token modifier that respects the Helmet Compoments.
```
Comment (hvanyo) [inline packages/vital-layout/src/components/Helmet/HelmetClean.tsx:86]:
```suggestion
 * A token modifier that respects the Helmet Components.
```

File: packages/vital-layout/src/components/Layout/LayoutClean.tsx [hunk 8]
```diff
@@ -83,8 +83,23 @@ export const LayoutCleanBase: FC<LayoutProps> = (layoutProps: LayoutProps) => {
...
+/**
+ * This is the base component for layout.
+ *
+ * @category Component
+ *
+ */
+const LayoutClean = designable(layoutComponents, 'Layout')(LayoutCleanBase);
 
+/**
+ * A token modifier that respects the Layout Compoments.
```
Comment (hvanyo) [inline packages/vital-layout/src/components/Layout/LayoutClean.tsx:95]:
```suggestion
 * A token modifier that respects the Layout Components.
```

File: packages/vital-layout/src/components/Logo/LogoClean.tsx [hunk 9]
```diff
@@ -53,14 +68,22 @@ const LogoBase: FC<LogoProps> = ({ components: C }) => (
 );
 
 /**
- * Crete a logo token.
+ * A token modifier that respects the Logo Compoments.
```
Comment (hvanyo) [inline packages/vital-layout/src/components/Logo/LogoClean.tsx:71]:
```suggestion
 * A token modifier that respects the Logo Components.
```

**Comment by dewen** [comment]
Smoke tested on items listed in description.

**Review by dewen — APPROVED** [review]

**Pull request merged**